src/Commands/Base.js

```diff
@@ -0,0 +1,113 @@
+'use strict'
+
+/**
+ * adonis-installer
+ * @license MIT
+ * @copyright Adonis - Harminder Virk <virk@adonisjs.com>
+*/
+
+const pify = require('pify')
+const which = require('which')
+const semver = require('semver')
+const Command = use('Adonis/Src/Command')
+const Spinner = require('cli-spinner').Spinner
+
+/**
+ * Base Command of the Adonis-CLI.
+ *
+ * @class Base
+ * @module Adonis
+ * @submodule cli
+ */
+class Base extends Command {
+
+  /**
+   * Constructor.
+   *
+   * @constructor
+   * @return {void}
+   */
+  constructor () {
+    super()
+
+    this.spinners = []
+  }
+
+  /**
+   * Check the Node.js version to ensure that it match the requirements.
+   *
+   * @protected
+   * @method _checkRequirements
+   * @return {void}
+   */
+  _checkRequirements () {
+    const nodeVersion = process.version.replace('v', '')
```

Comment:
Not required, `semver.satisfies` will remove the `v` itself. https://www.npmjs.com/package/semver#versions